lib/endpoint/find-parameters.js

```diff
@@ -216,3 +216,12 @@ function addPaginationParameters (state) {
     result.params.push(...PAGINATION_VARIABLES)
   })
 }
+
+function makeRelativeDefaultAbsolute (value) {
+  for (let i = 0; i < value.length; i++) {
+    if (value[i].default === 'Time.now') {
+      value[i].default = value[i].default.replace('Time.now', '<current date/time>')
+    }
+  }
+  return value
+}
```

Comment:
`makeRelativeDefaultAbsolute` is probably not a good name? Maybe do something like `replaceTimeNowDefault`? Also like in the other PR, passing in `state.results[0].params` is not enough, please pass in state like `findInBlocks` and the other methods above. Also please leave the empty line above the `// check for dublicate names` comment